src/App.js

```diff
@@ -21,18 +22,18 @@ export default function App() {
     <BrowserRouter>
       <AppContext.Provider value={appState}>
         <MainHeader />
-        <main className={styles.App}>
```

Comment:
I've added the `<main />` element to the `<PageContentMain />` component. In hindsight, do you think it's better to restore it to the `<App />` ?

Reply:
See next comment

Reply:
Yes,  I saw it. I like it in the PageContentMain better, but I am not sure if it will play nice with fullwidth content that we're going to need in the PageContentMain.